modules/acquia_cms_common/config/install/user.role.user_administrator.yml

```diff
@@ -6,9 +6,6 @@ label: 'User Administrator'
 weight: -4
 is_admin: null
 permissions:
-  - 'administer CAPTCHA settings'
```

Comment:
these permission got removed from here but not added conditional basis, please update accordingly.